doc/articles/using-uno-ui.md

```diff
@@ -392,3 +392,36 @@ If a view needs to keep the keyboard opened when tapping on it, use the `Uno.UI.
 
 ## Creating/Using Android Activities
 At the root of every Android Uno app, lies a `BaseActivity` class that extends from `Android.Support.V7.App.AppCompatActivity` which is part of the [Android v7 AppCompat Support Library](https://developer.android.com/topic/libraries/support-library/features.html#v7-appcompat). If you ever need to create a new Activity within your app or within Uno you must be sure to extend `BaseActivity` and, if you need to apply a Theme to the activity, ensure that the Theme you set is a `Theme.AppCompat` theme (or descendant).
+
+### Using the ILLinker for WebAssembly
+
+The [linker step](https://github.com/mono/linker/tree/master/docs) (also known as tree shaking, or IL Trimming) is responsible for the detection and removal of code that may not be used at runtime. This step is particularly important when targeting WebAssembly or native code in general, to reduce significantly the final package size.
+
+The linker is generally efficient at removing code but in some cases it may remove too much code, making some assemblies fail to work properly. In particular, assemblies or features that rely on reflection don't work by default with the linker enabled. `JSON.NET` is a good example of this, as it relies on reflection very heavily.
+
+In those cases, Uno provides access to the linker through the `LinkerConfig.xml` file in the Uno CrossPlatform project template.
+
+When parts of an application fail to work, you can:
+- Add full assembly names to the `LinkerConfig.xml` file:
+```xml
+<assembly fullname="MyAssembly" />
+```
+- Add namespaces for the linker to ignore:
+```xml
+  <assembly fullname="MyAssembly">
+	<!-- This is required by JSon.NET and any expression.Compile caller -->
+	<type fullname="MyNamespace*" />
+  </assembly>
+```
+
+As a troubleshooting step to determine if the linker is causing your code to break, you can also disable the linker completely by adding the following to your `csproj` file:
+```xml
+<PropertyGroup>
+  <WasmShellILLinkerEnabled>true</WasmShellILLinkerEnabled>
+</PropertyGroup>
+```
+Note that disabling the linker is not recommended as it can force the compiler to generate very large WebAssembly modules AOT process, and go over the browser's size limits.
```

Comment:
```suggestion
Note that disabling the linker is not recommended for production builds as it can force the compiler to generate very large WebAssembly modules during the AOT compilation process, and go over the browser's size limits.
```